package.xml

```diff
@@ -32,7 +32,9 @@
   <depend>libboost-dev</depend>
   <depend>libboost-system-dev</depend>
   <depend>libboost-regex-dev</depend>
-  <depend>octomap</depend>
+  <!-- Octomap is an optional dependency. There seems to be an issue resolving
+    this key on the ROS buildfarm, so disabling it for now -->
+  <!-- <depend>octomap</depend> -->
   <depend>lz4</depend>
```

Comment:
Note we are also adding a liblz4 key if you are just interested in using the library (BSD) versus the CLI (GPL) https://github.com/ros/rosdistro/pull/40263

Reply:
Okay, I can take a look at replacing it. I'll go ahead and merge this for now.